packages/math/init.lua

```diff
@@ -344,6 +359,18 @@ Finally, here is a little secret. This notation:
 
 \noindent In other words, the notation using \code{&} and \code{\\\\} is only a syntactic sugar for a two-dimensional array constructed with braces.
 
+\paragraph{Numbered equations}
+Equations can be numbered in display mode adding the options \autodoc:parameter{numbered} and \autodoc:parameter{counter}.
+When \autodoc:parameter{numbered=true}, the equations are numbered using a default "equation" counter.
+A different counter may be set by using the option \autodoc:parameter{counter=id}, and this setting will also enable numbering. 
+The default numbering format is "(xx)", but this style may be overriden by defining a custom \autodoc:command{\math:numberingstyle} function, e.g.
+
+\begin[type=autodoc:codeblock]{raw}
+\define[command=math:numberingstyle]{(\show-counter[id="equation"])}
```

Comment:
I am not fond of the example using a macro, because macros can't use options, so here this example would always use the "equation" counter regardless of what the user passed to the `\math` command. I'd suggest removing the example, and just expanding the description as follows (my changes in bold):

... this style may be overrid**d**en by defining a custom `\autodoc:command{\math:numberingstyle}` **command**. **The counter `id` is passed as parameter to this hook.**

(By the way, _overridden_ takes two -d-, I think)